challenge_manipulation/src/manipulation.py

```diff
@@ -45,7 +45,7 @@
 from empty_shelf_designator import EmptyShelfDesignator
 
 # PDF writer
-import pdf
+# import pdf
```

Comment:
Ouch, reviews already paying off. 
I outcommented that stuff because it was giving me errors while testing. Something with xhtml. 

The offending commit should be reverted.